src/main/httpCommandServer.ts

```diff
@@ -66,22 +73,24 @@ export class HttpCommandServer {
       const path = url.pathname;
       const pathParts = path.split('/');
 
+      console.log(`Processing request ${ method } ${ path }`);
       if (pathParts.shift()) {
-        response.writeHead(40, { 'Content-Type': 'text/plain' });
+        console.log(`400: Unexpected data in URL ${ path } before first slash.`);
```

Comment:
Hmm, does this need `console.log` or is `console.debug` enough? (That is: should we log in normal production environments?)

Same for the missing handler log, etc.

Reply:
Agreed, non-error situations should be `console.debug`ged.